packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -62,6 +62,16 @@ export class Wallet {
     return await this.client.call('getbalance', ['*', minimumConfirmation, includeWatchOnly], 'bignumber')
   }
 
+  /**
+   * Returns an object with all balances
```

Comment:
https://github.com/DeFiCh/jellyfish/pull/362/files#r654395003
@jingyi2811 add full stop here? @aikchun

Reply:
Nah it's fine LOL

For context:
https://github.com/DeFiCh/whale/pull/58#discussion_r629503905